test/TestRunner.py

```diff
@@ -20,11 +20,9 @@
 import os
 
 import pytest
-
```

Comment:
The problem here is the same as in other places: isort thinks that ansiblelint is third-party and puts it together with pytest.